glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
> rbd's lack of implementation was more of a oversight I think

No, I just dropped it intentionally because I see it as a duplicate of open and nothing more for rbd. I didn't know why we need it because open could fail then bailout. If we pass open, then we pass .check_config...